docs/src/prototypes/CopyToClipboard/CopyToClipboard.tsx

```diff
@@ -1,41 +1,25 @@
-import { Text, Tooltip, ShorthandValue, TextProps, TooltipProps } from '@stardust-ui/react'
+import { ShorthandValue, Tooltip, TooltipProps } from '@stardust-ui/react'
 import * as copyToClipboard from 'copy-to-clipboard'
 import * as _ from 'lodash'
 import * as React from 'react'
 
-import { NotificationContext } from './NotificationProvider'
+import { Notification, NotificationContext } from './NotificationProvider'
 
 export type CopyToClipboardProps = {
-  attached?: boolean
-  pointing?: boolean
+  tooltip?: ShorthandValue<TooltipProps>
+  attach?: boolean
```

Comment:
```suggestion
  attached?: boolean
```

Should it be `attached` again?